docs/source/references/cli_reference.rst

```diff
@@ -166,6 +166,15 @@ Manage stakes and other staker-related operations.
 |  ``--lock-until``       | Enable re-staking lock until release period |
 +-------------------------+---------------------------------------------+
 
+**Stake Rewards Command Actions**
+
++----------------------+-------------------------------------------------------------------------------+
+| Action               |  Description                                                                  |
++======================+===============================================================================+
+|  ``show``            | Show available and historical staking rewards                                 |
++----------------------+-------------------------------------------------------------------------------+
+|  ``withdraw``        | Withdraw staking compensation from the contract to your wallet                |
```

Comment:
I love the documentation changes <3  The command is much clearer now. Just a minor comment here:
```suggestion
|  ``withdraw``        | Withdraw unlocked tokens from the contract to your wallet                |
```